src/dotnet/APIView/APIViewWeb/Models/UsageSampleModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System.Security.Claims;
+
+namespace APIViewWeb
+{
+    public class UsageSampleModel
+    {
+        [JsonProperty("id")]
+        public string SampleId { get; set; } = IdHelper.GenerateId();
+        public string ReviewId { get; set; }
+        public int RevisionNum { get; set; }
+        public string UsageSampleFileId { get; set; }
+        public string UsageSampleOriginalFileId { get; set; }
```

Comment:
Why do we need   string fields for usage file IDs here? Shouldn't we be using a property of type `UsageSampleFileModel` here to store these file ID info? Where is `UsageSampleFileModel` used?